css/main.css

```diff
@@ -0,0 +1,117 @@
+*{
+    background-color: rgb(0, 0, 0);
+    font-family: "Roboto",sans-serif;
+    color: rgb(223, 48, 48);
+}
+.TB{
+    height: 12vw;
+}
+#camera{
+    display: inline;
+    padding-left:150px;
+}
+.orientation{
+    border:solid black;
+    padding: 0%;
+    margin-left: 52%;
+    padding-bottom: 30%;
+    width: 50vw;
+    float: right;
+}
+.timer{
+    font-size: 10vw;
+    width: 20vw;
+    height: 12vw;
+    margin: auto;
+    display: inline;
+    vertical-align: top;
+}
+.TBR2{
+    border:solid white;
+    padding-top: 3.5%;
+    padding-bottom: 3.5%;
+    margin-left:0.25%;
+    margin-right:0.25%;
+    font-size:210%;
+    float:right;
+}
+.hidden{
+    display: none;
+}
+.hiddengyro{
+    display: inline;
+    margin-right: 1.5vw;
+    width: 28vw;
+    height: auto;
+    float: right;

```

Comment:
warning: display:inline has no effect on floated elements (but may be used to fix the IE6 double-margin bug).